CIP-????/README.md

```diff
@@ -0,0 +1,394 @@
+---
+CIP: ????
+Title: NFT Metadata Update Oracles
+Status: Proposed
+Category: Metadata
+Authors:
+    - Nicolas Ayotte <nick@equine.gg>
+    - George Flerovsky <george@mlabs.city>
+    - Samuel Williams <samuel@mlabs.city>
+Implementors:
+    - Equine <[https://www.equine.gg/](https://www.equine.gg/)>
+    - MLabs <[https://mlabs.city/](https://mlabs.city/)>
```

Comment:
```suggestion
    - Equine <https://www.equine.gg/>
    - MLabs <https://mlabs.city/>
```
(markdown doesn't cause an error within the preamble fields, but it's not recognised either)

Reply:
Fixed